pkg/feature/builder.go

```diff
@@ -56,51 +60,40 @@ func Define(featureName string) *featureBuilder {
 	return fb
 }
 
-// TargetNamespace sets the namespace in which the feature should be applied.
-func (fb *featureBuilder) TargetNamespace(targetNs string) *featureBuilder {
-	fb.targetNs = targetNs
-
-	return fb
-}
-
 func (fb *featureBuilder) Source(source featurev1.Source) *featureBuilder {
 	fb.source = source
 
 	return fb
 }
 
-// Used to enforce that Manifests() is called after ManifestsLocation() in the chain.
-type requiresManifestSourceBuilder struct {
-	*featureBuilder
-}
+// TargetNamespace sets the namespace in which the feature should be applied.
+// If not set, the feature will be applied in the application namespace (where this operator lives).
```

Comment:
```suggestion
// If not set, the feature will be applied in the application namespace.
```

That was clearly a hallucination. Thanks for spotting